src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnHorseZelda;
 
+typedef void (*EnHorseZeldaActionFunc)(struct EnHorseZelda*, GlobalContext*);
+
 typedef struct EnHorseZelda {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x15C];
+    /* 0x014C */ s32 actionIndex;
```

Comment:
We normally just call this field `action`
```suggestion
    /* 0x014C */ s32 action;
```